index.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <script src="dist/app.js" defer></script>
-    <link rel="stylesheet" href="styles.css">
+    <script src="/dist/app.js" defer></script>
```

Comment:
Absolute path to resolve correctly when the URL is in a sub-folder.

Problem:
`/shop/foobar` with `<script src="dist/app.js"></script>` fails to load `/shop/dist/app.js`

Solution:
`/shop/foobar` with `<script src="/dist/app.js"></script>` successfully loads `/dist/app.js`